### DISCUSSION TITLE
k8s部署Doris可以用于生产吗

### DISCUSSION THREAD

**Discussion by lgq19991225** [apache/doris#24577] [A - General / Q&A]
k8s部署Doris可以用于生产吗，使用nfs或GlusterFS作为存储可行吗

**Comment by yagagagaga** [comment]
如果你想用 k8s 部署 Doris，可以加我微信，拉你进内测群 ![](https://user-images.githubusercontent.com/21151126/222130177-a85c1e2b-bdaa-4c96-98bc-2c4c8da02615.JPG)
